docs/proposals/automated-node-removal.md

```diff
@@ -0,0 +1,60 @@
+# Proposal: Automation of node removal procedure
+
+Last updated: 24.06.2021
+
+## Abstract
+
+Implementation of the option to delete CSI CRs from a node, which going to be removed. 
+
+## Background
+
+User manual actions for node removal are described in [doc](https://github.com/dell/csi-baremetal/blob/master/docs/node-removal.md). The goal of this POC is automating most of the steps of that procedure. 
+
+## Proposal
+
+CSI CRs deletion(Volumes, AC, ACR, Drives, LVGs) will be triggerred on deletion of appropriate csibmnode and we be performed by node-controller.
+Before implementation, we need to replace node-uuid annotation with label and add node-affinity `uuid label is exists` to csi-baremetal-node daemonset.
+
+Updated node removal steps performed by user:
+1. Get node uuid mapped to node name
```

Comment:
User should just put annotation on the node.

Reply:
updated